### PR TITLE
sg-content-box additional spacings

### PR DESCRIPTION
**Pull request by aju**
screen shot 2017-04-04 at 15 15 22

### CODE REVIEW

**Comment by burakukula** [comment]
@aju maybe we could replace this with just text?
screen shot 2017-04-04 at 15 23 33

File: src/components/content-box/content-box.html [hunk 1]
```diff
@@ -516,7 +526,19 @@ <h2 class="sg-header-secondary">This is a title for context box</h2>
 
 <section class="docs-block">
     <aside class="docs-block__info">
-        <h3 class="docs-block__header">Spaced-top elements inside</h3>
+      <h3 class="docs-block__header">
+        Spaced-top elements inside<br/>
```
Comment (kdzwinel) [inline src/components/content-box/content-box.html]: why you have a `br` here but don't have one above?

Comment (aju) [inline src/components/content-box/content-box.html]: fixed

**Comment by aju** [comment]
screen shot 2017-04-04 at 15 26 59

**Comment by aju** [comment]
screen shot 2017-04-04 at 15 37 00

**Pull request merged**